pytorch_lightning/core/optimizer.py

```diff
@@ -180,7 +180,10 @@ def _init_optimizers_and_lr_schedulers(model: "pl.LightningModule") -> Tuple[Lis
         optim_conf = _MockOptimizer()
 
     optimizers, lr_schedulers, optimizer_frequencies, monitor = _configure_optimizers(optim_conf)
-    lr_schedulers = _configure_schedulers(lr_schedulers, monitor, not model.automatic_optimization)
+    if model.automatic_optimization:
+        lr_schedulers = _configure_schedulers_automatic_opt(lr_schedulers, monitor)
+    else:
+        lr_schedulers = _configure_schedulers_manual_opt(lr_schedulers, monitor)
```

Comment:
```suggestion
    _configure = _configure_schedulers_automatic_opt if model.automatic_optimization else _configure_schedulers_manual_opt
    lr_schedulers = _configure(lr_schedulers, monitor)
```
just fancier but not really needed :]